client-frameworks-support/client-support-ui5/src/README.md

```diff
@@ -0,0 +1,144 @@
+# Luigi Client UI5 Support Library
+​
+The Luigi Client UI5 Support Library offers a set [features](#features) which make it easier to use the [Luigi micro frontend framework](https://luigi-project.io/) with UI5 applications. 
+​
+## How to use the library 
+​
+1. Import the library in your `package.json` file:
+```javascript
+npm install @luigi-project/client-support-ui5
+```
+​
+2. Once the library is imported and saved in your UI5 application, you need to add `ComponentSupport` to the `index.html` as well as to allow the `data-sap-ui-frameOptions`.
+```javascript
+<script id="sap-ui-bootstrap" src="resources/sap-ui-core.js" data-sap-ui-resourceroots='{
+				"com.sap.luigiclient": "./"
```

Comment:
Maybe we don't call the demoapp `com.sap.luigiclient` - this may confuse the consumers

Reply:
sth like luigi.ui5-demoapp